src/model/guild/guild_id.rs

```diff
@@ -568,6 +568,30 @@ impl GuildId {
         http.as_ref().edit_member(self.0, user_id.0, &map)
     }
 
+    /// Disconnects a member from a voice channel in a guild.
```

Comment:
```suggestion
    /// Disconnects a member from a voice channel in the guild.
```
The a/an article would be appropriate here if the user were to pass a guild id to the method. However, since we're already working on a guild id, it implies which guild the member will be disconnected in.